src/screens/Dashboard/CompactTickets.tsx

```diff
@@ -0,0 +1,141 @@
+  const [now, setNow] = useState<number>(Date.now());
+  useInterval(() => setNow(Date.now()), 1000);
+
+  const {fareContracts, customerProfile} = useTicketState();
+  const activeFareContracts = filterActiveOrCanBeUsedFareContracts(
+    fareContracts,
+  ).sort(function (a, b): number {
+    const isA = isValidRightNowFareContract(a);
+    const isB = isValidRightNowFareContract(b);
+
+    if (isA === isB) return 0;
+    if (isA) return -1;
+    return 1;
+  });
+
+  // TODO: Double check if this is how it should be implemented, using the customer profile!
+  const hasActiveTravelCard = !!customerProfile?.travelcard;
+  const hasEnabledMobileToken = useHasEnabledMobileToken();
+  const {
+    deviceIsInspectable,
+    isError: mobileTokenError,
+    fallbackEnabled,
+  } = useMobileTokenContextState();
+  const {tariffZones, userProfiles, preassignedFareproducts} =
+    useFirestoreConfiguration();
+  const {t, language} = useTranslation();
+
+  return (
+    <>
+      <ThemeText
+        type="body__secondary"
+        color="secondary"
+        style={itemStyle.sectionText}
+        accessibilityLabel={t(TicketsTexts.header.title)}
+      >
+        {t(TicketsTexts.header.title)}
+      </ThemeText>
+      {activeFareContracts?.map((fareContract, index) => {
+        const travelRights =
+          fareContract.travelRights.filter(isPreactivatedTicket);
+        const firstTravelRight = travelRights[0];
+        const {fareProductRef: productRef, tariffZoneRefs} = firstTravelRight;
+        const ticketIsInspectable = isInspectableTicket(
+          firstTravelRight,
+          hasActiveTravelCard,
+          hasEnabledMobileToken,
+          deviceIsInspectable,
+          mobileTokenError,
+          fallbackEnabled,
+        );
+        const fareContractState = fareContract.state;
+        const {startDateTime, endDateTime} = firstTravelRight;
+        const validTo = endDateTime.toMillis();
+        const validFrom = startDateTime.toMillis();
+        const validityStatus = getValidityStatus(
+          now,
+          validFrom,
+          validTo,
+          fareContractState,
+        );
+        const [firstZone] = tariffZoneRefs;
+        const [lastZone] = tariffZoneRefs.slice(-1);
+        const fromTariffZone = findReferenceDataById(tariffZones, firstZone);
+        const toTariffZone = findReferenceDataById(tariffZones, lastZone);
+        const preassignedFareProduct = findReferenceDataById(
+          preassignedFareproducts,
+          productRef,
+        );
+        const userProfilesWithCount = mapToUserProfilesWithCount(
+          travelRights.map((tr) => tr.userProfileRef),
+          userProfiles,
+        );
+
+        return (
+          <Sections.Section withPadding>
+            <Sections.GenericClickableItem
+              onPress={() =>
+                onPressDetails && onPressDetails(fareContract.orderId)
+              }
```

Comment:
Very nit-picky here, but I think this can be written simpler:

```suggestion
              onPress={() => onPressDetails?.(fareContract.orderId)}
```